pkg/resources/preprocessor/workloads.go

```diff
@@ -70,8 +70,13 @@ processors:
     log_statements:
     - context: log
       statements:
-      - set(attributes["COMM"], attributes["_COMM"])
-      - delete_matching_keys(attributes, "^_.*")
+      - merge_maps(cache, body, "upsert") where not IsString(body) and attributes["filename"] == nil
```

Comment:
How can the body not be a string? What else can it be?

Reply:
Ah, it can be a map, right?

Reply:
In the case of journald logs it is a map.  That is specifically the case we want to handle.